atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -204,29 +213,61 @@ public CassandraClientPool(CassandraKeyValueServiceConfig config) {
 
     private CassandraClientPool(CassandraKeyValueServiceConfig config, StartupChecks startupChecks) {
         this.config = config;
-        config.servers().forEach(this::addPool);
         refreshDaemon = Tracers.wrap(PTExecutors.newScheduledThreadPool(1, new ThreadFactoryBuilder()
                 .setDaemon(true)
                 .setNameFormat("CassandraClientPoolRefresh-%d")
                 .build()));
-        refreshDaemon.scheduleWithFixedDelay(() -> {
-            try {
-                refreshPool();
-            } catch (Throwable t) {
-                log.error("Failed to refresh Cassandra KVS pool."
-                        + " Extended periods of being unable to refresh will cause perf degradation.", t);
+        this.startupChecks = startupChecks;
```

Comment:
honestly we should clean up this constructor to take a ScheduledExecutorService for refreshDaemon and inject that from factory method & tests